cmd/tracee-ebpf/main.go

```diff
@@ -158,6 +163,26 @@ func main() {
 				return fmt.Errorf("error creating Tracee: %v", err)
 			}
 
+			if listenMetrics {
+				err := metrics.RegisterEbpfEngineMetrics(t)
+				if err != nil {
+					if debug {
+						fmt.Fprintf(os.Stderr, "Error registering prometheus metrics: %v\n", err)
```

Comment:
I think, errors should always be printed.

Reply:
All other errors in the main.go file are only printed on debug mode